modules/mri_violations/test/mri_violationsTest.php

```diff
@@ -154,6 +154,7 @@ public function setUp()
              'series_description' => 'Test Description',
              'SeriesUID'          => '5555',
              'TarchiveID'         => '263',
+             'MriProtocolGroupID' => 1,
```

Comment:
The MriProtocolGroupID might not be 1 if the auto_increment is altered. I would be safer to get the Id first based on the Name.  Unless you ignore the previous comment:

> 
> The `1` should be generated by the auto_increment. And, the insert at line 830 should use a subselect WHERE Name = 'Default MRI protocol checks group' to get this ID
> 
> _Originally posted by @xlecours in https://github.com/aces/Loris/pull/4321/files#diff-49c955f1af7f1ad35cdd7a9aceb590d1R811